src/main/java/io/sudhakar/student/controller/RestTemplateController.java

```diff
@@ -0,0 +1,76 @@
+package io.sudhakar.student.controller;
+
+import io.sudhakar.student.dto.Employee;
+import io.sudhakar.student.dto.ServiceResponse;
+import io.sudhakar.student.dto.StudentResponse;
+import io.sudhakar.student.service.impl.RestTemplateServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping("/employee")
+public class RestTemplateController {
+    private final RestTemplateServiceImpl restTemplateService;
+
+    public RestTemplateController(RestTemplateServiceImpl restTemplateService) {
+        this.restTemplateService = restTemplateService;
+    }
+
+    @GetMapping
+    public ResponseEntity<StudentResponse<?>> getAllEmployee(@RequestParam("urlName") String urlName) {
+
+        log.info("api = /employee, method = GET, status = IN_PROCESS");
+
+        ServiceResponse<?> serviceResponse = restTemplateService.getAllEmployee(urlName);
+
+        log.info("api = /employee, method = GET, status = SUCCESS");
+        return ResponseEntity.status(serviceResponse.getHttpStatus()).body(new StudentResponse<>(serviceResponse.getData()));
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<StudentResponse<?>> getEmployeeById(@RequestParam("urlName") String urlName, @PathVariable int id) {
```

Comment:
same comment as above for all the methods

Reply:
hard coded for all methods in RestTemplateServiceImpl class